pkg/domain/infra/tunnel/images.go

```diff
@@ -367,3 +363,23 @@ func (ir *ImageEngine) Shutdown(_ context.Context) {
 func (ir *ImageEngine) Sign(ctx context.Context, names []string, options entities.SignOptions) (*entities.SignReport, error) {
 	return nil, errors.New("not implemented yet")
 }
+
+func (ir *ImageEngine) Scp(ctx context.Context, args []string, parentFlags []string, quiet bool) error {
```

Comment:
```suggestion
func (ir *ImageEngine) Scp(ctx context.Context, source, destination string, parentFlags []string, quiet bool) error {
```
Rather than hide source and destination in args, call them out by name.

Reply:
for the native implementation I need to put them back in an array for parsing if I make this change.

I will change the name of the array here to make it more obvious what we are dealing with is that ok?